fibheap/Documentation/FibonacciHeap.java

```diff
@@ -0,0 +1,580 @@
+ /***********************************************************************
```

Comment:
Is this here purely as a reference example?  If so, you might just put a link to this source code in the documentation and not include the file.